src/components/modals/TransferPDNTModal/TransferPDNTModal.tsx

```diff
@@ -151,8 +153,16 @@ function TransferPDNTModal({
                 setIsValidAddress(validity)
               }
               validationPredicates={{
-                [VALIDATION_INPUT_TYPES.ARWEAVE_ID]: (id: string) =>
-                  arweaveDataProvider.validateArweaveId(id),
+                [VALIDATION_INPUT_TYPES.ARWEAVE_ID]: {
+                  fn: (id: string) => arweaveDataProvider.validateArweaveId(id),
+                },
+                [VALIDATION_INPUT_TYPES.ARWEAVE_ADDRESS]: {
+                  fn: (id: string) =>
+                    arweaveDataProvider.validateArweaveAddress(id),
+                  required: false,
+                  message:
```

Comment:
i wasn't clear, this should be the failed state message. i.e. `The address does not appear to be a valid Arweave address.`

you can change it to `errorMessage` to be more clear